syntax = "proto3";



message PlayerSkillPBComponent {
  uint64 id = 1; 
  uint32 skill_table_id = 2; 
}

message PlayerSkillListPBComponent {
  repeated PlayerSkillPBComponent skill_list = 1;
}